### ISSUE TITLE
Incorrect handling of directly created compound intervals

### ISSUE THREAD

**Issue by Wally869** [Wally869/MidiStructurer#2]
**Issue Description:**
When creating an Interval from several ones, by using the Intervals field in the Interval constructor, the string representation is incorrect.
Seems like I hadn't worked out the calculations for compounding intervals: intervals given will be kept as they were given and not simplified.

**Code sample:**
```python
>>> i=Interval(Intervals=[Interval(5, "Perfect"), Interval(5, "Perfect")])
>>> i
Interval(1 Octaves + Interval(5-Perfect--7 semitones))
>>> i.Intervals
[Interval(5-Perfect--7 semitones), Interval(5-Perfect--7 semitones)]
```

**Comment by Wally869** [comment]
**Proposed Solution:**  
Add a function to compute a simplified Interval from the intervals given as input.  

Fixing this issue should be useful to analyze more complex Intervals structure.

**Comment by Wally869** [comment]
Fixed in c7b4d4684c68799561f78c66c049e9ba38886fe7

Previous implementation for compound intervals relied on the Interval.FromNotes method as the way to create compound intervals, since I wasn't sure how to handle them (it didn't quite make sense to choose a quality for a compound interval, since it would be dependent which note it is applied to).

New implementation will check all intervals provided in input to sum the tonal distances. Then it will output a compound interval creating a minor/major/perfect interval depending on the tonal distance (after  adjusting for the octaves). In case no valid interval is found from this, this means the created interval is a tritone so it is set as such (Interval(5 - Diminished)).
